recipe/meta.yaml

```diff
@@ -1,17 +1,29 @@
-{% set version = "14.2.3" %}
+{% set version = "14.2.4" %}
+{% set sha_hash = "55c780f7faa5c8cabd83ebbb84b68e5e0e09732de70a129f6b3c801e905415dd" %}
+{% set build_number = "0" %}
+
 
 package:
   name: reproc
   version: '{{ version }}'
 
 source:
   - url: https://github.com/DaanDeMeyer/reproc/archive/v{{ version }}.tar.gz
-    sha256: ea87db7a780a81bc27bce3e6c8e5f31b831fb8a1d06c6b5c2e1c2554d1c55e00
+    sha256: {{ sha_hash }}
     patches:
       - apple_gettime.patch  # [osx]
 
 build:
-  number: 0
+  number: {{ build_number }}
+  missing_dso_whitelist:  # [s390x]
+    - $RPATH/ld64.so.1    # [s390x]  Something is slightly amiss in the s390x build process.
```

Comment:
I think this happens because conda-build does not understand that the linker is part of the sysroot.